source/slang/slang-capabilities.capdef

```diff
@@ -224,9 +224,10 @@ def hlsl_nvapi : hlsl;
 /// [Version]
 def hlsl_2018 : _sm_5_1;
 
+// https://devblogs.microsoft.com/directx/cooperative-vector : HLSL coopvec requires sm 6.9 now.
 /// Represent compatibility support for the deprecated POC DXC
 /// [Version]
-def hlsl_coopvec_poc : _sm_6_8;
+def hlsl_coopvec_poc : _sm_6_9;
```

Comment:
the official support for coopvec is available in sm_6_9, but this capability is for the nvidia-specific implementation (requires nvidia branch of dxc), so it is correct to assume sm_6_8.